### PR TITLE
Added NSAttributedString and Screenshots 

### PR DESCRIPTION
**Pull request by samuelbeek**

### CODE REVIEW

File: cheat_sheet.swift [hunk 1]
```diff
@@ -218,5 +218,34 @@ extension UIColor {
...
+// use Avenir Light 28pt font with 6pt lineSpacing
+
+// set the properties
+let font = UIFont(name: "Avenir-Light", size: 28)
+let style = NSMutableParagraphStyle()
+style.lineSpacing = 6
+
+let attributes = [NSParagraphStyleAttributeName : style, NSFontAttributeName: font!]
+
+var attributedString = NSAttributedString(string: "Example text", attributes:attributes)
```
Comment (zackshapiro) [inline cheat_sheet.swift]: Maybe a note here about the example text or how you can leave that blank?

File: cheat_sheet.swift [hunk 2]
```diff
@@ -218,5 +218,34 @@ extension UIColor {
...
+
+// 16) Make a Screenshot 
+
+func makeScreenshot() -> UIImage {
+    
+    UIGraphicsBeginImageContext(self.view.bounds.size);
+    
+    self.view.layer.renderInContext(UIGraphicsGetCurrentContext())
+    var viewImage : UIImage = UIGraphicsGetImageFromCurrentImageContext()
```
Comment (zackshapiro) [inline cheat_sheet.swift]: remove the space after viewImage (styling)

File: cheat_sheet.swift [hunk 3]
```diff
@@ -218,5 +218,34 @@ extension UIColor {
...
+
+let attributes = [NSParagraphStyleAttributeName : style, NSFontAttributeName: font!]
+
+var attributedString = NSAttributedString(string: "Example text", attributes:attributes)
+
+// 16) Make a Screenshot 
+
+func makeScreenshot() -> UIImage {
+    
+    UIGraphicsBeginImageContext(self.view.bounds.size);
```
Comment (zackshapiro) [inline cheat_sheet.swift]: remove `;`

File: cheat_sheet.swift [hunk 4]
```diff
@@ -218,5 +218,34 @@ extension UIColor {
...
+style.lineSpacing = 6
+
+let attributes = [NSParagraphStyleAttributeName : style, NSFontAttributeName: font!]
+
+var attributedString = NSAttributedString(string: "Example text", attributes:attributes)
+
+// 16) Make a Screenshot 
+
+func makeScreenshot() -> UIImage {
+
```
Comment (zackshapiro) [inline cheat_sheet.swift]: remove new line

File: readme.md [hunk 5]
```diff
@@ -17,8 +17,13 @@ These are things that I lookup frequently or solutions I spent some time on. I h
...
 * Add a person to AddressBook
 * Use HexColors codes for UIColor 
+* Create a NSAttributedString 
+* Make a Screenshot
 * More to come
 
-Created by [@ZackShapiro](http://twitter.com/zackshapiro)
+Created and Curated by [@ZackShapiro](http://twitter.com/zackshapiro)
+
+Additions by: [@SamuelBeek](http://twitter.com/samuelbeek)
```
Comment (zackshapiro) [inline readme.md]: :+1:

**Comment by zackshapiro** [comment]
@samuelbeek will you rebase this on master please?

**Comment by samuelbeek** [comment]
Yep! I'll dive into this later this week!

—
Samuel Beek,  I make digital stuff.


If something is urgent, call me: +31 6 21 11 10 11


If you’re in the us: +1 (347) 5693227.

**Comment by zackshapiro** [comment]
Thanks man

Best,
Zack

**Comment by samuelbeek** [comment]
I think they're al resolved now :).

**Comment by zackshapiro** [comment]
@samuelbeek I'm not able to merge this in. I think you need to fetch the upstream master and rebase your branch on it to be merge-able

**Comment by samuelbeek** [comment]
That’s weird; I’ll look into this today.


—
Samuel Beek,  I make digital stuff.


If something is urgent, call me: +31 6 21 11 10 11


If you’re in the us: +1 (347) 5693227. 






Twitter.

**Comment by zackshapiro** [comment]
Hey @samuelbeek, sorry to bug you about this, wanted to get your changes merged in. Would you mind rebasing? LMK if you need help pulling from the upstream branch or anything

**Comment by samuelbeek** [comment]
Sorry Zack, I forgot! I’ll rebase it tommorow :)


—
Samuel Beek,  I make digital stuff.


If something is urgent, call me: +31 6 21 11 10 11





Twitter.

**Pull request merged**